src/plugins/interactive-media/interactive-media.yml

```diff
@@ -40,3 +40,12 @@ addAnswer:
 markCorrectAnswer:
   en: Mark as correct answer
   de: Als richtige Antwort markieren
+continue:
+  en: continue
+  de: fortfahren
+replay:
+  en: replay
+  de: wieder spielen
```

Comment:
"noch einmal abspielen" or "wiederholen" would sound better. Also, I think we have all button labels start with a capital letter (in both languages), so these three labels have a little bit of an "unfamiliar look".

Reply:
True, changed to "Wiederholen" and capitalized first letter in both languages, thanks!